### PR TITLE
Fixes #19148 - Add description field to subnets

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

**Review by timogoebel — APPROVED** [review]
@tbrisker : Thanks. Code looks fine and feature works well! Ready to merge when tests are 💚 .

File: app/views/subnets/_fields.html.erb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <%= text_f f, :name %>
+<%= text_f f, :description %>
```
Comment (ohadlevy) [inline app/views/subnets/_fields.html.erb]: Text area or field?

Comment (tbrisker) [reply app/views/subnets/_fields.html.erb]: right, fixed.

Comment (timogoebel) [reply app/views/subnets/_fields.html.erb]: Oh, I thought that was on purpose. :-)

**Comment by ares** [comment]
would you mind also opening hammer-cli-foreman PR? Should be similar to https://github.com/theforeman/hammer-cli-foreman/pull/260

**Comment by ares** [comment]
and last idea - would it make sense to make this available in [ENC output](https://github.com/theforeman/foreman/blob/develop/app/models/subnet.rb#L18)? fair to say no :-)

EDIT: [safe mode](https://github.com/theforeman/foreman/blob/develop/app/models/subnet.rb#L95) should be allowed though

**Review by timogoebel — CHANGES_REQUESTED** [review]
@tbrisker : I think, @ares has a point here. Could you please allow safe mode and add this to enc output while you're at it. Can't hurt.

**Comment by tbrisker** [comment]
@ares @timogoebel Thanks, added to enc and safemode (though not sure there will be much use for it) and created pr for hammer as well

**Comment by timogoebel** [comment]
@tbrisker : One thing I found:

image

But I think, this is unrelated to this PR.

**Review by timogoebel — APPROVED** [review]
@tbrisker : Thanks, looks good.

**Comment by tbrisker** [comment]
@timogoebel is that after you stretched it or is that how it loaded for you?

**Comment by timogoebel** [comment]
@tbrisker: It's after I changed the size. It looks good when the page loads.

**Review by ares — APPROVED** [review]
Thanks, I have no more comments :+1:

**Pull request merged**